nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/GetMongoTest.java

```diff
@@ -135,15 +135,15 @@ public void testValidators() {
         // invalid projection
         runner.setVariable("projection", "{a: x,y,z}");
         runner.setProperty(GetMongo.QUERY, "{a: 1}");
-        runner.setProperty(GetMongo.PROJECTION, "${projection}");
+        runner.setProperty(GetMongo.PROJECTION, "{a: z}");
```

Comment:
I can't immediately tell the reason for this change, since the projection is changed from x,y,z to z. Is this on purpose?

Reply:
The test was reporting a false positive for me until I made that change. It seemed to think that x,y,z was a valid projection, but when I changed it to a:z it recognized it was invalid. Since none of that is valid JSON, I'm not even sure why any of it is working instead of the Mongo client API throwing an exception on parsing it.